docs/cloud/serverless.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Serverless 
+layout: default
+parent: Cloud
+has_children: false
+nav_order: 14 
+---
+
+# Serverless
+*Currently in preview*
+
+## Why Serverless?
+Serverless is a new architecture to FeatureBase that separates compute power and storage. This allows for independent scaling of compute, storage, or both.  
+
+## Creating a Serverless Database
+Before you begin, you should check out how to create a database in our cloud enviornment, as there are a lot of similarities. ([See here](/docs/cloud/cloud-databases/cloud-db-create/))  
+One of the major differences is Serverless databases don't have a database shape!
```

Comment:
No exclamation marks in help. You're informing a user how to perform tasks not having a conversation.

Additionally, this is only relevant if the user knows what a database shape is. 

My recommendation: delete this text.